schedule/yast/releasenotes_origin+unregistered_aarch64.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: releasenotes_origin+unregistered
+description: >
+  Test fate#323273 - Check the origin (rpm or url) of the showed release notes.
+vars:
+  EXIT_AFTER_START_INSTALL: '1'
+  SCC_REGISTER: 'none'
+  YUI_REST_API: 1
+schedule:
+  registration:
+    - installation/registration/skip_registration
+  extension_module_selection:
+    - installation/module_selection/select_module_desktop
+  add_on_product:
+    - installation/add_on_product_installation/accept_add_on_installation
+  release_notes:
```

Comment:
this hook doesn't exit and it shouldn't exit in the default yaml, because it is not a phase of the installation, we need to incorporate it to some other phase.
you can put it inside `system_role` before the gnome selection.